code/modules/cargo/blackmarket/blackmarket_items/emergency.dm

```diff
@@ -0,0 +1,52 @@
+/datum/blackmarket_item/emergency
+	category = "Emergency"
+
+/datum/blackmarket_item/emergency/plasma
```

Comment:
Fair. The upfront asking price is pretty brutal. The intent here was that you were getting strong-armed and gouged by the black market dealers since you had no better options.